swatch-metrics-hbi/src/main/resources/application.properties

```
@@ -118,10 +118,7 @@ quarkus.log.handler.splunk.format=%d %-5p [%c{3.}] (%t) %s%e%n
 quarkus.log.handler.splunk.include-exception=${SPLUNK_HEC_INCLUDE_EX:false}
 
 # otel config
-quarkus.otel.sdk.disabled=true
-%ephemeral.quarkus.otel.sdk.disabled=true
-%stage.quarkus.otel.sdk.disabled=false
-%prod.quarkus.otel.sdk.disabled=false
+quarkus.otel.sdk.disabled=${OTEL_DISABLED:true}
 
 quarkus.otel.exporter.otlp.endpoint=${OTEL_EXPORTER_OTLP_ENDPOINT:http://splunk-otel-collector:4317}
```

```suggestion
quarkus.otel.exporter.otlp.endpoint=${OTEL_EXPORTER_OTLP_ENDPOINT:http://localhost:4317}
```

The endpoint should be localhost:4317 which is the port that is listening the otel collector sidecar.